packages/@react-aria/table/stories/useTable.stories.tsx

```diff
@@ -244,3 +246,133 @@ export const TableWithSomeResizingFRsControlled = {
+      {...otherProps}>
+      <TableHeader columns={cols}>
+        {column => (
+          <Column allowsResizing key={column.uid} width={widths.get(column.uid)}>
+            {column.name}
+          </Column>
+        )}
+      </TableHeader>
+      <TableBody items={defaultRows}>
+        {item => (
+          <Row>
+            {columnKey => <Cell>{item[columnKey]}</Cell>}
+          </Row>
+        )}
+      </TableBody>
+    </DocsTable>
+  );
+}
+
+function AsyncSortTable() {
+  let list = useAsyncList({
+    async load({signal}) {
+      let res = await fetch('https://swapi.py4e.com/api/people/?search', {
+        signal
+      });
+      let json = await res.json();
+      return {
+        items: json.results
+      };
+    },
+    async sort({items, sortDescriptor}) {
+      return {
+        items: items.sort((a, b) => {
+          let first = a[sortDescriptor.column];
+          let second = b[sortDescriptor.column];
+          let cmp = (parseInt(first, 10) || first) < (parseInt(second, 10) || second)
+            ? -1
+            : 1;
+          if (sortDescriptor.direction === 'descending') {
+            cmp *= -1;
+          }
+          return cmp;
+        })
+      };
+    }
+  });
+
+  return (
+    <DocsTable
+      aria-label="Example table with client side sorting"
+      sortDescriptor={list.sortDescriptor}
+      onSortChange={list.sort}>
+      <TableHeader>
+        <Column key="name" allowsSorting allowsResizing>Name</Column>
+        <Column key="height" allowsSorting allowsResizing>Height</Column>
+        <Column key="mass" allowsSorting allowsResizing>Mass</Column>
+        <Column key="birth_year" allowsSorting allowsResizing>Birth Year</Column>
+      </TableHeader>
+      <TableBody items={list.items}>
+        {(item: any) => (
+          <Row key={item.name}>
+            {(columnKey) => <Cell>{item[columnKey]}</Cell>}
+          </Row>
+        )}
+      </TableBody>
+    </DocsTable>
+  );
+}
+
+export const DocExample = {
+  args: {},
+  render: (args) => (
+    <DocsTable
+      aria-label="Table with selection"
+      selectionMode="multiple"
+      onColumnResizeStart={action('onColumnResizeStart')}
+      onColumnResizeEnd={action('onColumnResizeEnd')}
+      onResizeStart={action('onResizeStart')}
+      onResize={action('onResize')}
+      onResizeEnd={action('onResizeEnd')}
+      {...args}>
+      <TableHeader columns={columns}>
+        {column => (
+          <Column allowsResizing key={column.uid}>
+            {column.name}
+          </Column>
+        )}
+      </TableHeader>
+      <TableBody items={defaultRows}>
+        {item => (
+          <Row>
+            {columnKey => <Cell>{item[columnKey]}</Cell>}
+          </Row>
+        )}
+      </TableBody>
+    </DocsTable>
+  )
+};
+
+export const DocExampleControlled = {
+  args: {columns: columnsFR},
+  render: (args) => (
+    <ControlledDocsTable {...args} />
+  )
+};
+
+export const DocExampleWithSorting = {
```

Comment:
is this actually supposed to sort? how do I do it?

Reply:
Ah, I'll need to update that example, relic of an earlier stash. Will dig into the resizer disappearing on mouse down, might be something I can do in Resizer for that

Reply:
ok, feel free to make a meeting with me if it's taking a while